src/Server/Server.fs

```diff
@@ -510,11 +499,32 @@ let azureAccount() = task {
         return Error $"{errorType}: {error.Message}"
 }
 
+let getDefaultSubscription() = task {
+    let! account = azureAccount() |> Async.AwaitTask
```

Comment:
Inside `task { ... }` blocks, you don't need to use `Async.AwaitTask` because using `let!` already unwraps the `T` from `Task<T>` 

You only need `Async.AwaitTask` you are converting from `Async<T>` to `Task<T>` but in this case, `azureAccount()` already returns `Task<T>` so this is redundant